docs/docs/install/environment-variables.md

```diff
@@ -51,16 +51,13 @@ These environment variables are used by the `docker-compose.yml` file and do **N
 | `PORT`                  | Web Port              | `3000`  | web              |
 | `SERVER_PORT`           | Server Port           | `3001`  | server           |
 | `MICROSERVICES_PORT`    | Microservices Port    | `3002`  | microservices    |
-| `MACHINE_LEARNING_PORT` | Machine Learning Port | `3003`  | machine learning |
 
 ## URLs
 
 | Variable                          | Description                  |                Default                | Services              |
 | :-------------------------------- | :--------------------------- | :-----------------------------------: | :-------------------- |
 | `IMMICH_WEB_URL`                  | Immich Web URL               |       `http://immich-web:3000`        | proxy                 |
 | `IMMICH_SERVER_URL`               | Immich Server URL            |      `http://immich-server:3001`      | web, proxy            |
-| `IMMICH_MACHINE_LEARNING_ENABLED` | Enabled machine learning     |                `true`                 | server, microservices |
-| `IMMICH_MACHINE_LEARNING_URL`     | Immich Machine Learning URL, | `http://immich-machine-learning:3003` | server, microservices |
```

Comment:
There's a `MACHINE_LEARNING_HOST` env in the ML service used in addition to `MACHINE_LEARNING_PORT`, but it seems it's not listed on this page. It doesn't make sense to change it from the default `0.0.0.0` if running in a container, but for native deployments it could be useful.

Reply:
I added that too in the "Ports" section - it seemed the right place although the section is called "Ports"... maybe we should rename it to something like "Ports/Bindings" or something?

Reply:
Yeah, that's be good